server/portal/apps/projects/serializers.py

```diff
@@ -55,7 +55,10 @@ def default(self, obj):  # pylint: disable=method-hidden, arguments-differ
                     if related is not None:
                         ret[attname] = _seralize_user(related)
                 else:
-                    ret[attname] = field.value_to_string(obj)
+                    if val == None:
```

Comment:
works well but what about the model?  

`description = models.TextField(blank=True, null=True)`; is it possible that the `null` is the problem there.

Reply:
Doesn't that just meant he field is nullable? The alternative is to set a default description and eliminate null=True and blank=True (which i seem to recall have their own issues in Django.)

Reply:
Hmm, what if we just set the default to an empty string?

Reply:
this is fine now from my end. Joon Yee and I discussed this and i understand that maybe not touching the model for now might be a good approach when migrating.

Reply:
so this or a change or model field are good for me 👍

Reply:
Ok 👍